src/coreclr/nativeaot/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -1169,6 +1169,8 @@ REDHAWK_PALEXPORT bool PalGetMaximumStackBounds(_Out_ void** ppStackLowOut, _Out
         status = pthread_attr_get_np(thread, &attr);
 #elif HAVE_PTHREAD_GETATTR_NP
         status = pthread_getattr_np(thread, &attr);
+#elif defined(HOST_WASM) && defined(FEATURE_WASM_MANAGED_THREADS)
+        // We dont have a pthread_getattr_np, but so far we don't need it.
```

Comment:
Do we not? Could you test whether our Emscripten version already has a working version of it, it needs `_GNU_SOURCE` defined to be available in the headers.